lib/Core/ErrorState.h

```diff
@@ -66,6 +66,8 @@ class ErrorState {
 
   ref<Expr> retrieveStoredError(ref<Expr> address) const;
 
+  bool isInStoredError(ref<Expr> address);
```

Comment:
@Himeshi isInStoreError() better be made private and const, as it does not change the ErrorState object, as well as not used outside of ErrorState.